pkg/services/featuremgmt/registry.go

```diff
@@ -1823,6 +1823,14 @@ var (
 			Owner:        grafanaPluginsPlatformSquad,
 			FrontendOnly: false,
 		},
+		{
+			Name:         "unifiedNavbars",
+			Description:  "Enables unified navbars",
+			Stage:        FeatureStageGeneralAvailability,
+			Owner:        grafanaPluginsPlatformSquad,
+			FrontendOnly: true,
+			Expression:   "true", // enabled by default
```

Comment:
will switch this to false before merge, just to help review & ephemeral instances for now